app/components/each-property/component.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+export function getPropertyInputType(property) {
```

Comment:
This is where we make assumptions about what kind of `schema-field` should be bound to this property.  We could optionally add a new property attribute (outside of JSON schema spec) that defines which component to use.  That seems more flexible, honestly.